lib/registries/addon/components/license-viewer/template.hbs

```diff
@@ -1,30 +1,32 @@
 <OsfDialog as |dialog|>
-    <dialog.trigger>
-        <OsfButton
+    <div local-class='LicenseView'>
+        <dialog.trigger>
+            <OsfButton
             data-analytics-name='View license'
             local-class='LinkButton'
             @type='link'
             @onClick={{dialog.open}}
-        >
-            {{@registration.license.name}}
-        </OsfButton>
-    </dialog.trigger>
+            >
+                {{@registration.license.name}}
```

Comment:
I suspect this line is what you want to wrap in your local-class to minimize overreaching with the css.

Reply:
Got it, thank you Brian. Ok, I'm locally testing these changes and will resubmit this PR.